test/Core.Test/OrganizationFeatures/OrganizationCollections/DeleteCollectionCommandTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿
+using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.OrganizationFeatures.OrganizationCollections;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Test.AutoFixture.OrganizationFixtures;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.OrganizationFeatures.OrganizationConnections;
+
+[SutProviderCustomize]
+public class DeleteCollectionCommandTests
+{
+
+    [Theory, BitAutoData]
+    [OrganizationCustomize]
+    public async Task DeleteAsync_DeletesCollection(Collection collection, SutProvider<DeleteCollectionCommand> sutProvider)
+    {
+        // Act
+        await sutProvider.Sut.DeleteAsync(collection);
+
+        // Assert
+        await sutProvider.GetDependency<ICollectionRepository>().Received().DeleteAsync(collection);
+        await sutProvider.GetDependency<IEventService>().Received().LogCollectionEventAsync(collection, EventType.Collection_Deleted);
+    }
+
+    [Theory, BitAutoData]
+    [OrganizationCustomize]
+    public async Task DeleteManyAsync_DeletesManyCollections(Collection collection, Collection collection2, SutProvider<DeleteCollectionCommand> sutProvider)
+    {
+        // Arrange
+        var collectionIds = new[] { collection.Id, collection2.Id };
+
+        sutProvider.GetDependency<ICollectionRepository>()
+            .GetManyByManyIdsAsync(collectionIds)
+            .Returns(new List<Collection> { collection, collection2 });
+
+        // Act
+        await sutProvider.Sut.DeleteManyAsync(collectionIds);
+
+        // Assert
+        await sutProvider.GetDependency<ICollectionRepository>().Received()
+            .DeleteManyAsync(Arg.Is<IEnumerable<Guid>>(ids => ids.SequenceEqual(collectionIds)));
+
+        await sutProvider.GetDependency<IEventService>().Received().LogCollectionEventAsync(collection, EventType.Collection_Deleted, Arg.Any<DateTime>());
```

Comment:
Here's an example if you update the event service to use bulk collection events:
https://github.com/bitwarden/server/blob/9ea72832db72bc8056ee61516ec324be343091b8/test/Core.Test/OrganizationFeatures/Groups/DeleteGroupCommandTests.cs#L42-L45